pkg/app/master/commands/xray/cli.go

```diff
@@ -134,6 +135,8 @@ var CLI = cli.Command{
 			doHashData = true
 		}
 
+		doTarUtf8 := ctx.Bool(FlagHashData)
```

Comment:
this might be a typo... otherwise it's not clear why the "hash-data" flag is repurposed here